pubspec.yaml

```diff
@@ -11,9 +13,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_map: ^1.0.0
-  flutter_map_marker_popup: ^2.2.0
+  flutter_map: ^2.0.0
   latlong2: ^0.8.1
+  flutter_map_marker_popup:
+    git:
+      url: https://github.com/Feynallein/flutter_map_marker_popup.git
```

Comment:
pls dont use git url

Reply:
I had to use this in order to have access to flutter map. Since flutter_map_marker_popup is now updated it's not required anymore.